Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php

```diff
@@ -22,7 +23,14 @@ class SubprocessProjectionCatchUpCommandController extends CommandController
      * @Flow\Inject
      * @var ContentRepositoryRegistry
      */
-    protected $contentRepositoryRegistry;
+    protected ContentRepositoryRegistry $contentRepositoryRegistry;
```

Comment:
With that type hint, this is no longer a lazy injection.
Totally fine with me, but in that case I would suggest to replace it with a constructor injection for good measure :)

```php
public function __construct(
    private readonly ContentRepositoryRegistry $contentRepositoryRegistry,
) {
    parent::__construct();
}
```

Reply:
> seems a rather big little pitfall

Yes, I agree and that's why I try to avoid this everywhere.. It also makes the code pure PHP which feels cleaner (even though this class will always be a Flow class)